Typos in S2

ammedmar opened the pull request.

ammedmar committed:
S2 remove double blank spaces

ammedmar committed:
S2 replace " }" with "}" and same for "{"

ammedmar committed:
S2 remove blank first line

ammedmar committed:
S2 remove multi blank lines and indent envs

ammedmar committed:
S2 replace ":" by " \colon " in function defs

ammedmar committed:
S2 add missing blank lines between envs

ammedmar committed:
S2 make display eqs have own line

ammedmar committed:
S2 split long lines to have one sentence per line

ammedmar committed:
S2 typo, "material is to Joyce"

ammedmar committed:
S2 replace overused word "atlas"

ammedmar committed:
S2 add comment about possible extra superscript

% anibal: ...

ammedmar committed:
S2 add comment about using \mapsto for points

ammedmar committed:
S2 add comment about "ordinarily transverse"

ammedmar committed:
S2 add "involved" to "independent of the choices"

ammedmar committed:
S2 add blank line after lemma

ammedmar committed:
S2 typo "neighborood"

ammedmar committed:
S2 typo "anologous"

ammedmar committed:
S2 add "results" for clarity

ammedmar committed:
S2 typo "progess"

ammedmar committed:
S2 replace math in text in math for just math

ammedmar committed:
S2 prevent unfortunate line break

ammedmar committed:
S2 typo, wrong capitalization

ammedmar committed:
S2 flag clunky sentence and propose alternative

ammedmar committed:
S2 fix indentation and add "~" for good measure

ammedmar committed:
S2 improve readability

ammedmar committed:
S2 add comma and "of them" for readability

ammedmar committed:
S2 add comma

ammedmar committed:
S2 add suggestion for homogeneous style

ammedmar committed:
S2 add missing dot to i.e.

ammedmar committed:
S6 add missing dot to i.e.

ammedmar committed:
To-do add two general suggestions

devsinha reviewed (commented):
I am leaving a comment.

devsinha commented on parts/s6.tex:
```diff
@@ -442,7 +442,7 @@ \subsubsection{Dualization in cubes}\label{S: dual cubes}
 \item If $i\in F_{01}$ (i.e.\ the coordinate $x_i$ is free in $F$), then $x_i=1/2$ in $F^\vee$.
 
-\item If $i\in F_0$ (i.e\ the coordinate $x_i$ is bound to $0$ in $F$), then $x_i$ is free in $[0,1/2]$ in $F^\vee$.
+\item If $i\in F_0$ (i.e.\ the coordinate $x_i$ is bound to $0$ in $F$), then $x_i$ is free in $[0,1/2]$ in $F^\vee$.
```
Here is a comment associated to line 445.

gf2078 committed:
Fixed the superscript issue Anibal noted

gf2078 committed:
Added the "\mapsto"  Anibal wanted

There will be more of these - feel free to change them yourself

gf2078 committed:
Added further comment about "ordinary" transverse

gf2078 committed:
Fixed sentence Anibal found clunky

gf2078 committed:
Merge branch 'GBF-S2' into S2-had to merge to TeX properly

gf2078 committed:
Fixed plural

gf2078 committed:
Added foundations.pdf to gitignore

gf2078 committed:
Removed to-do items that I already did; fixed a typo

gf2078 commented:
That didn't work. How do I make this thing go away?

gf2078 commented:
What happened to my last comment?

gf2078 commented:
Nevermind - somehow moved on to a different pull request. 
Still not sure how to dismiss them, though

gf2078 committed:
Updated ordinary transversality to plain transversality

Also fixed a few "co-ordinate"s to "coordinate

gf2078 committed:
Changed some multitransversality stuff

Move the remark about n-ary transversality not implying iterated transversality for maps of manifolds, and added a proposition showing that some transversality assumptions still imply others. I think this will be needed for working on showing that the the intersection map takes the geometric cup product to the cubical cup product.

gf2078 committed:
Merge branch 'GBFtransversalitylemma' into S2

gf2078 committed:
Deleted temporary files

gf2078 reviewed (approved):
Look's good
